Another C++20 issue building binaryen on macOS-13

From the emsdk CI:

```
/Users/distiller/project/binaryen/main/src/passes/StringLifting.cpp:77:47: error: no member named 'view' in 'std::stringstream'
        importedStrings[global->name] = wtf16.view();
                                        ~~~~~ ^
1 error generated.
make[2]: *** [CMakeFiles/binaryen.dir/src/passes/StringLifting.cpp.o] Error 1
```

--- a/src/passes/StringLifting.cpp
+++ b/src/passes/StringLifting.cpp
@@ -74,6 +74,7 @@ struct StringLifting : public Pass {
         if (!valid) {
           Fatal() << "Bad string to lift: " << wtf8;
         }
+        // TODO: Use wtf16.view() once we have C++20.
         importedStrings[global->name] = wtf16.str();
         found = true;
       }

--- a/src/passes/StringLifting.cpp
+++ b/src/passes/StringLifting.cpp
@@ -74,7 +74,7 @@ struct StringLifting : public Pass {
         if (!valid) {
           Fatal() << "Bad string to lift: " << wtf8;
         }
-        importedStrings[global->name] = wtf16.view();
+        importedStrings[global->name] = wtf16.str();
         found = true;
       }
     }

--- a/src/support/json.cpp
+++ b/src/support/json.cpp
@@ -39,7 +39,7 @@ void Value::stringify(std::ostream& os, bool pretty, int indent) {
       [[maybe_unused]] bool valid =
         wasm::String::convertWTF8ToWTF16(wtf16, getIString().view());
       assert(valid);
-      wasm::String::printEscapedJSON(os, wtf16.view());
+      wasm::String::printEscapedJSON(os, wtf16.str());
       return;
     }
     case Array: {

--- a/src/support/json.cpp
+++ b/src/support/json.cpp
@@ -39,6 +39,7 @@ void Value::stringify(std::ostream& os, bool pretty, int indent) {
       [[maybe_unused]] bool valid =
         wasm::String::convertWTF8ToWTF16(wtf16, getIString().view());
       assert(valid);
+      // TODO: Use wtf16.view() once we have C++20.
       wasm::String::printEscapedJSON(os, wtf16.str());
       return;
     }

